src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../../TodosContext/TodosContext';
+import { FilteringBy } from '../../types/FilteringBy';
+
+export const Footer: React.FC = () => {
+  const { todos, handleClearCompleted, filteringBy, setFilteringBy } =
+    useContext(TodosContext);
```

Comment:
Create a hook to avoid passing context every time:
`const useTodos = () => useContext(TodosContext);`